docs/design/manipulation/architecture.med

```diff
@@ -0,0 +1 @@
+(TODO)
```

Comment:
This can be added in another PR

Reply:
.med -> .md

Reply:
Looks like a duplicate of `architecture.md`

Reply:
sorry for that. Removed in 6b5e82dcca8e79d835e0e4cfd148901ca473298b